source/win-main/DocumentTabs.vue

```diff
@@ -435,15 +454,16 @@ body.darwin {
 
   &.dark {
     div#tab-container {
-      border-bottom-color: rgb(11, 11, 11);
+      border-top-color: rgb(11, 11, 11);
```

Comment:
Again, we need to make sure that this doesn't break the Windows and macOS styles